aten/src/ATen/native/SummaryOps.cpp

```diff
@@ -64,4 +65,56 @@ _bincount_cpu(const Tensor& self, const Tensor& weights, int64_t minlength) {
   });
 }
 
+Tensor& histc_out(Tensor& hist, const Tensor &self, int64_t nbins, Scalar minvalue, Scalar maxvalue) {
+  if (nbins <= 0) {
+    AT_ERROR("bins must be > 0");
+  }
+
```

Comment:
Please TOCK_CHECK dtype of hist, as anything can be used in `out=` version.

Reply:
Done